cmd/event/list/list.go

```diff
@@ -63,27 +60,12 @@ func NewCommand() *cobra.Command {
 			} else {
 				url = config.Conf.Clients["CoreData"].Url() + clients.ApiEventRoute
 			}
-			resp, err := http.Get(url)
+			eventList := eventList{}
```

Comment:
remove `eventList{}` structure and use `var eventList []models.Event` instead. (or just `events` as shorter name).
Sending eventsList{} structure to utils.ListHelper function returns error: `json: cannot unmarshal array into Go value of type list.eventList`